integration_testing/features/super-admin/super-admin-setting-port.feature

```diff
@@ -0,0 +1,29 @@
+Feature: Super admin set the kolibri server port
+    Super admin should be able to set the koliri server port into different ports
+
+  Background:
+    Given That I have or don't have an options.ini file with an alternate port set
+      And I have or don't have the KOLIBRI_HTTP_PORT environment variable set
+
+  Scenario: Setting the port with the options.ini file exist
```

Comment:
Please re-phrase to: `Scenario: Setting the port inside an existing options.ini file`